### PR TITLE
Errors in FF browser console #673

### PR DESCRIPTION
**Pull request by atropnikov**
#673

### CODE REVIEW

File: Extension/lib/content-script/preload.js [hunk 1]
```diff
@@ -79,12 +79,13 @@
      * Initializing content script
      */
     var init = function () {
-        initRequestWrappers();
 
         if (!isHtml()) {
             return;
         }
 
+        initRequestWrappers();
```
Comment (Mizzick) [inline Extension/lib/content-script/preload.js]: We need request wrappers on `about:` pages as well.

File: Extension/lib/content-script/content-utils.js [hunk 2]
```diff
@@ -27,6 +27,13 @@
         return;
     }
 
+    /**
+     * It appears on the extension startup
```
Comment (ameshkov) [inline Extension/lib/content-script/content-utils.js]: Change to: `On extension startup contentPage is undefined`

File: Extension/lib/content-script/preload.js [hunk 3]
```diff
@@ -126,6 +127,16 @@
     };
 
     /**
+     * Checks location's protocol.
+     * Returns true if we are on a 'http(s)' or 'about:' page
```
Comment (ameshkov) [inline Extension/lib/content-script/preload.js]: It makes sense to have a better description of why do we need it on about: page.

This is due to the circumvention with using iframe's contentWindow

File: Extension/lib/content-script/preload.js (hunk 1 above)
Comment (atropnikov) [inline Extension/lib/content-script/preload.js]: Hm, but `about:` page is `HTMLDocument`, isn't it?

Comment (Mizzick) [inline Extension/lib/content-script/preload.js]: yeap, you are right.

**Pull request merged**